src/custom-elements/src/components/date-picker/date-picker.tsx

```diff
@@ -94,7 +94,7 @@ export class DatePicker {
   /**
    * Date change value event
    */
-  @Event({ eventName: 'chiDateChange' }) eventChange: EventEmitter;
+  @Event({ eventName: 'chiDatePickerChange' }) eventChange: EventEmitter;
```

Comment:
The event name `chiDateChange` in `chi-date-picker` was causing the `handleDateChange` function to trigger twice because it's the same name used in the event emission from `chi-date`. Changing the event name of `chi-date-picker` to something unique solves this issue. Also having duplicate event name seems like an anti-pattern and we should avoid it?